pkgs/tools/admin/awscli2/default.nix

```diff
@@ -67,6 +67,7 @@ with py.pkgs; buildPythonApplication rec {
   postPatch = ''
     substituteInPlace setup.cfg \
       --replace "colorama>=0.2.5,<0.4.4" "colorama" \
+      --replace "cryptography>=3.3.2,<37.0.0" "cryptography" \
```

Comment:
This maybe not needed, not sure the actual status in 22.05 branch, but if it compiles, it compiles!